index.js

```diff
@@ -179,10 +179,14 @@ module.exports = function() {
       .then(function(results) {
         var output = results[results.length - 1]; // compilation output is always last
 
-        if (output.kind == 'success') {
+        if (output.kind === 'success') {
           alreadyCompiledFiles.push(resourcePath);
           callback(null, output.result);
         } else {
+          if (typeof output.error === 'string') {
+            output.error = new Error(output.error);
+          }
```

Comment:
Not sure if this is a good idea, @eeue56 please let us know what you think.

I'm not sure if it has to be addressed in `node-elm-compiler` 😕